### PR TITLE
Update LazyComponentsResolver.php

### PR DESCRIPTION
**Pull request by Skaywalker**
Resolve in findVueFiles files path replace \\ directory separator to /

### CODE REVIEW

**Comment by Skaywalker** [comment]
Resolve in findVueFiles files path replace \\\ directory separator to /

**Comment by innocenzi** [comment]
@Skaywalker is there an issue with that PR? I haven't taken the time to test it yet, that's why it didn't get merged with last release.

**Comment by Skaywalker** [comment]
There was no problem with it, I just accidentally closed it. I haven't experienced any problems. right, replace only runs when it finds //

File: packages/laravel/src/Architecture/LazyComponentsResolver.php [hunk 1]
```diff
@@ -228,7 +228,7 @@ protected function findVueFiles(string $directory, string $baseDirectory, null|s
                 if (str($path)->endsWith($this->getExtensions())) {
                     $files[] = [
                         'namespace' => $namespace,
-                        'path' => str($path)->replaceStart(base_path(), '')->ltrim('/\\')->toString(),
+                        'path' => str($path)->replaceStart(base_path(), '')->ltrim('/\\')->replace('\\', '/')->toString(),
```
Comment (Tofandel) [inline packages/laravel/src/Architecture/LazyComponentsResolver.php:231]:
```suggestion
                        'path' => str($path)->replaceStart(base_path(), '')->replace('\\', '/')->ltrim('/')->toString(),
```

**Comment by innocenzi** [comment]
For some reason `git` kept rejecting me pulling this PR so I just committed directly and added you both as co-authors.

**Pull request closed without merging**